06_gpu_and_ml/comfyui/comfyapp.py

```diff
@@ -72,126 +48,105 @@
     .run_commands(  # use comfy-cli to install the ComfyUI repo and its dependencies
         "comfy --skip-prompt install --nvidia"
```

Comment:
minor note: i don't think there's a way to pin which version of comfyui you're installing, which represents some risk

Reply:
Ah I was under the impression the comfy version was pinned by the comfy-cli version but maybe that's not the case